File: chapter3/RaiseToPower.java
import java.util.Scanner;

public class RaiseToPower{


	public static void main(String[]args){

		Scanner scanner = new Scanner(System.in);
		
		System.out.print("Enter first number: ");
		double num1 = scanner.nextDouble();

		System.out.print("Enter second number: ");
		double num2 = scanner.nextDouble();

		int count = 1;
		int count2 = 0;
		double result = num1;
		while(count < num2){
			num1 = num1;
			result *= num1;
			count++;

		}
		
		System.out.printf("%.0f raise to power %.0f is %.0f",num1,num2,result);

 
		
			



}

}

File: chapter3/raise_to_power.py

num1 = int(input("Enter first number: "));
num2 = int(input("Enter second number: "));
count = 1;
result = num1;


while(count < num2):
	result *= num1;
	count += 1;

print(f"{num1} raise to power {num2} is {result}");

 